libtextsecure/protobufs.js

```diff
@@ -4,15 +4,11 @@
     window.textsecure.protobuf = {};
 
     function loadProtoBufs(filename) {
-        return dcodeIO.ProtoBuf.loadProtoFile({root: 'protos', file: filename}, function(error, result) {
-           if (error) {
-              throw error;
-           }
-           var protos = result.build('textsecure');
-           for (var protoName in protos) {
-              textsecure.protobuf[protoName] = protos[protoName];
-           }
-        });
+        var result = dcodeIO.ProtoBuf.loadProtoFile({root: 'protos', file: filename});
```

Comment:
This triggers a warning in the console output.
```
Synchronous XMLHttpRequest on the main thread is deprecated because of its detrimental effects to the end user's experience. For more help, check https://xhr.spec.whatwg.org/.
```